tests/client_test.go

```diff
@@ -161,7 +161,6 @@ func TestStreamClient(t *testing.T) {
 	}
 }
 
-// Test adding the slow_start property on an upstream server
 func TestStreamUpstreamServerSlowStart(t *testing.T) {
```

Comment:
`TestStreamUpstreamServerSlowStart`, `TestUpstreamServerMaxConns`, `TestUpstreamServerMaxFails`, `TestUpstreamServerFailTimeout`, `TestUpstreamServerSlowStart` are almost the same. Can we combine them into a single test similar to `TestUpstreamServerDefaultParameters`? 

The same applies to the stream upstream parameters